client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/hooks.tsx

```diff
@@ -108,3 +109,57 @@ export function useResendVerificationCode(): {
 		},
 	} );
 }
+
+export function useContactModalTitleAndSubtitle(
+	type: 'email' | 'phone',
+	action: AllowedMonitorContactActions
+): {
+	title: string;
+	subtitle: string;
+} {
+	const translate = useTranslate();
+
+	const getContactModalTitleAndSubTitle = useMemo(
+		() => ( {
+			email: {
+				add: {
+					title: translate( 'Add new email address' ),
+					subtitle: translate( 'Please use only your number or one you have access to.' ),
```

Comment:
I haven't looked at the designs for this so apologies if I'm off base, but the subtitle doesn't make sense to me in the context of email.  what about using 'email address' or something similar in place of 'number'?